### DISCUSSION TITLE
Welcome to DSA Discussions!

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
 * If you need Galaxy brain badge then answer 2 of the following questions and I will mark them correct. After 2nd correct marked answer you will be getting a galaxy brain badge💪💪💪.

**Comment by Preetiraj3697** [comment]
Thankyou for discuss I want to Galaxy Brain badge. I answer all question. I hope I get galaxy brain badge get soon.
